c/models/utils.h

```diff
@@ -21,18 +21,40 @@
 //------------------------------------------------------------------------------
 #ifndef dt_MODELS_UTILS_h
 #define dt_MODELS_UTILS_h
-#include <memory>
 #include <limits>
+#include <memory>
+#include <numeric>
+#include "datatable.h"
 
 template <typename T>
 using tptr = typename std::unique_ptr<T[]>;
 using uint64ptr = std::unique_ptr<uint64_t[]>;
 using sizetptr = std::unique_ptr<size_t[]>;
 using sizetvec = std::vector<size_t>;
 
-
 void calculate_coprimes(size_t, std::vector<size_t>&);
 
+
+/*
+*  Create list of sorting indexes.
+*/
+template <typename T>
+sizetvec sort_index(const std::vector<T> &v) {
+
+  // Initialize original index locations
+  sizetvec index(v.size());
+  iota(index.begin(), index.end(), 0);
```

Comment:
better use `std::iota` and `std::sort` here, otherwise some compilers will complain